SDBlockDevice.cpp

```diff
@@ -147,6 +147,11 @@
 #include "mbed_debug.h"
 #include <errno.h>
 
+/* Required version: 5.5.4 and above */
+#if (MBED_VERSION < MBED_ENCODE_VERSION(5,5,4))
```

Comment:
Shouldn't this check also adds a condition if MBED_VERSION is defined ? Otherwise this won't compile for master (no version provided there).